src/Symfony/Component/Console/Event/ConsoleTerminateEvent.php

```diff
@@ -35,20 +33,16 @@ public function __construct(Command $command, InputInterface $input, OutputInter
 
     /**
      * Sets the exit code.
```

Comment:
let's remove that too :)

Reply:
all those changes would be for 4.x as we also removed other irrelevant phpdocs there. so it shouldn't be part of this PR or?

Reply:
for this pr it's fine to me, partial removals are less useful than full removals to me

Reply:
done